pkgs/development/python-modules/faker/default.nix

```diff
@@ -33,6 +33,10 @@ buildPythonPackage rec {
     substituteInPlace setup.py --replace "pytest>=3.8.0,<3.9" "pytest"
   '';
 
+  preCheck = ''
+    find tests -type d -name "__pycache__" | xargs rm -r
```

Comment:
Doing `-exec rm -r "{}"` fails on me so the next best thing.

Reply:
That should be moved to `postPatch`.